code/modules/mob/emote.dm

```diff
@@ -67,10 +73,10 @@
 			if (istype(M, /mob/abstract/observer))
 				messagemobs_neardead += M
 				continue
-			else if (istype(M, /mob/living) && !(type == 2 && (sdisabilities & DEAF || ear_deaf)))
+			else if (istype(M, /mob/living) && !(type == 2 && ((sdisabilities & DEAF || ear_deaf) && !hearing_aid)))
```

Comment:
```suggestion
			else if (istype(M, /mob/living) && !(type == 2 && ((sdisabilities & DEAF) && !hearing_aid) || ear_deaf))
```

If you want to make it so that:
- Hearing aid only works if they have permanent DNA deafness, you could do this to this and all the similar checks?

So we only care about `sdisabilities & DEAF` if we don't have a hearing_aid, else we only care about ear_deaf.

Having `sdisabilities & DEAF` implies we have `ear_deaf` so I guess we don't need to check for that in the same check?

Reply:
I'm a bit worried that this'll lead to stuff like ear_deaf being less than one in some cases, and people being able to hear still. I'm going to keep it as-is currently, though I might revisit it in the future